tests/resources/transaction/test_mining.py

```diff
@@ -30,6 +30,7 @@ def test_get_block_template_with_address(self):
                 'conflict_with': [],
                 'voided_by': [],
                 'twins': [],
+                'validation': 'initial',  # FIXME: change to 'full' when validations are enabled
```

Comment:
Will this test fail when validations are enabled?

Reply:
No. And actually I think 'full' wouldn't make any sense for a block that wasn't even mined yet. I'll remove that comment.